src/tests/indexTest.js

```diff
@@ -0,0 +1,21 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+describe('Get index route', () => {
+  it('Shows a welcome message when users visit our landing page', (done) => {
+    chai.request(app)
+      .get('/')
+      .end((error, res) => {
+        if (error) return error;
+        expect(res.status).to.equal(200);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.keys('status', 'message');
+        expect(res.body.status).to.a('string');
+        expect(res.body.message).to.a('string');
+        done();
+      });
+  });
+});
```

Comment:
We have controllers file, model file, and routes. Please follow the same convention and create the test files under the tests folder. We expect to see; `tests/controllers/indexTest.js, tests/model/indexTest.js, tests/routes/indexTest.js, tests/routes/usersTest.js`. In each of these files, you could come up with a basic test that does something like; ` it('True should be equal to true', (done) => {
    assert.equal('true', 'true');
    done();
  });` The goal here is just for us to see that when we run `npm test` the test runs and also passess.

Reply:
Also pay attention to the title of your PR, it should be something similar to the title of this PR here https://github.com/andela/phoenix-bn-backend/pull/6

Reply:
> We have controllers file, model file, and routes. Please follow the same convention and create the test files under the tests folder. We expect to see; `tests/controllers/indexTest.js, tests/model/indexTest.js, tests/routes/indexTest.js, tests/routes/usersTest.js`. In each of these files, you could come up with a basic test that does something like;
> 
> ` it('True should be equal to true', (done) => { assert.equal('true', 'true'); done(); });`
> 
> The goal here is just for us to see that when we run `npm test` the test runs and also passess.

Alright, I will do that

Reply:
> Also pay attention to the title of your PR, it should be something similar to the title of this PR here #6

Noted, I will that.